docs/installation.rst

```diff
@@ -19,6 +19,28 @@ You can also install it into a new virtual environment:
    $ source ~/.virtualenvs/cernopendata-client/bin/activate
    $ pip install cernopendata-client
 
+If you would like to use a more performing **pycurl** library for downloading data, please add the `pycurl` flavour while installing `cernopendata-client`:
+
+.. code-block:: console
+
+   $ pip install cernopendata-client[pycurl]
+
+Note that you need to have `curl <https://curl.se/download.html>`_ and `openssl <https://www.openssl.org/source/>`_ set up on your system.
```

Comment:
Perhaps just link to the homepages (rather than download pages) of the three tools?  

Chances are people have installed curl andd openssl already, so they may rather be interested to see what this XRootD stuff is all about, so rather seeing the home page instead of the download page.

(And nobody would use curl download to install it; people would use their package manager.)

P.S. While amending this, can you please look at the py3.7 test CI failure? Why did it fail?

Reply:
Done 👍